src/kakarot/instructions/system_operations.cairo

```diff
@@ -317,3 +390,81 @@ namespace CallHelper {
         return ctx;
     }
 }
+
+namespace CreateHelper {
+    // @notice Deploy a new Contract account and initialize a sub context at these addresses
+    //         with bytecode from calling context memory.
+    // @return The pointer to the updated calling context.
+    func create_sub_context{
```

Comment:
this naming sounds like you're creating a sub_context rather than it being a sub_context for the CREATE opcode
maybe `sub_context_create` could help? or even `sub_context_create_opcode`

Reply:
!! the CREATE kw is confusing indeed !!
however here I intended to actually say create like initialize, not create like CREATE. We're actually creating a sub context for CREATE (🤔)
Bottom line: `initialize_sub_context` may be better and somehow symmetrical with `finalize_calling_context` in terms of prefix/suffix